css/sign.css

```diff
@@ -0,0 +1,55 @@
+
+body {font-family: Arial, Helvetica, sans-serif; width: 50%; margin-left: 25%; margin-right: 25%; }
+form {border: 3px solid #f1f1f1; background-color: #94ffff  ;}
```

Comment:
Make use of your color scheme, don't use custom colors

Reply:
done